applications/jupyter-hpc-mpi/app.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "jupyter-hpc-mpi",
+    "version": "1.0.1",
+    "description": "Run an interactive Jupyter session with ability to launch mpi jobs.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "SINGULARITY",
+    "runtimeVersion": null,
+    "runtimeOptions": [
+        "SINGULARITY_RUN"
+    ],
+    "containerImage": "docker://debian:bullseye-slim",
```

Comment:
Could use `docker://alpine:3` for an even smaller no-op image

Reply:
> Could use `docker://alpine:3` for an even smaller no-op image

Yes, I started with alpine. But, alpine fails to run with this error:
```
INFO:    Creating SIF file...
Error relocating /opt/apps/xalt/xalt/lib64/libxalt_init.so: __strdup: symbol not found
```
interestingly, when I use idev, it runs fine. There are no bind mounts in the app definition. I'm assuming some libs are bound automatically when running via the slurm job. No, error with debian, so choose the safer option.

Reply:
Interesting, good choice 👍